### DISCUSSION TITLE
Need Max Payne 3 NPUB30902 v1.10 cheat update

### DISCUSSION THREAD

**Discussion by mfsi-prateeksharma** [bucanero/ArtemisPS3#14] [Cheat requests]
Hi @bucanero, 
I tried a lot to create a cheat for this game but no success.
Could you add a file for this?

**Comment by bucanero** [comment]
Sorry but I don’t have time to create codes

You can find about how to make codes here: https://github.com/Dnawrkshp/NetCheatPS3

Here's a video tutorial for NetCheat made by @Dnawrkshp: https://www.youtube.com/watch?v=2dxuoLTA_Lk
